creed/settings.py

```diff
@@ -0,0 +1,10 @@
+REGION_PARAM_FIELDS = ['region_start_x', 'region_start_y', 'fov_num_x', 'fov_num_y',
+                       'x_fov_size', 'y_fov_size', 'region_rand']
+MICRON_TO_STAGE_X_MULTIPLIER = 0.001001
+MICRON_TO_STAGE_X_OFFSET = 0.3116
+MICRON_TO_STAGE_Y_MULTIPLIER = 0.001018
+MICRON_TO_STAGE_Y_OFFSET = 0.6294
+STAGE_TO_PIXEL_X_MULTIPLIER = 1 / 0.06887
+STAGE_TO_PIXEL_X_OFFSET = 27.79
+STAGE_TO_PIXEL_Y_MULTIPLIER = 1 / -0.06926
+STAGE_TO_PIXEL_Y_OFFSET = -77.40
```

Comment:
Would a TMA mounted on a slant affect the fiducial constants or are these safe?

Reply:
Nope, the TMA is mounted sideways on the slide, but the fiducials are still on the slide in the same place